lighty-core/lighty-controller/src/test/resources/DataInitInvalidInitConfigJson.json

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
You do not need to specify all of the configuration that is not relevant for this test case. Missing values from json/xml configurations will be set to default.